scripts/test-dssspell-forge.sh

```diff
@@ -21,11 +21,11 @@ export DAPP_LIBRARIES="src/DssSpell.sol:DssExecLib:$DSS_EXEC_LIB"
 export DAPP_BUILD_OPTIMIZE=0   # forge turns on optimizer by default
 
 if [[ -z "$MATCH" && -z "$BLOCK" ]]; then
-    forge test --fork-url "$ETH_RPC_URL" --force
+    forge test --fork-url "$ETH_RPC_URL" --force -vvv
```

Comment:
default mode is not verbose, in case of failure `match` should be used to inspect trace

Reply:
fixed